src/main/java/com/growup/todoary/controller/TodoController.java

```diff
@@ -0,0 +1,23 @@
+package com.growup.todoary.controller;
+
+import com.growup.todoary.domain.Todo;
+import com.growup.todoary.service.TodoService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import java.util.List;
+
+@Controller
+@RequiredArgsConstructor
+public class TodoController {
+
+    private final TodoService todoService;
+
+    @GetMapping("/todos")
+    public ResponseEntity<List<Todo>> getTodos() {
+        return new ResponseEntity<>(todoService.getTodos(), HttpStatus.OK);
```

Comment:
정적 메소드를 사용해서 표현할 수 있겠네요
```suggestion
        return ResponseEntity.ok(todoService.getTodos());
```

Reply:
정적 메소드를 사용하도록 바꾸겠습니다!